extra/import_data.py

```diff
@@ -157,7 +164,7 @@ def main():
                                 # name contains patient number of parents
                                 # TODO: implement this properly
                                 if key.startswith("father") or key.startswith("mother") \
-                                    or key.startswith("brother") or key.startswith("sister"):
+                                        or key.startswith("brother") or key.startswith("sister"):
```

Comment:
Instead of `\` you can also use `()`.
```python
if (
    key.startswith("father") or key.startswith("mother") or 
    key.startswith("brother") or key.startswith("sister")
):
```

Reply:
I prefer to use \ here. It is simply what I am used to. unless of course it is better to use your suggestion according to the python style rules?

Reply:
I think both are correct methods, just wanted to give this as an example.